data/campaigns/Winds_of_Fate/scenarios/12_Epilogue.cfg

```diff
@@ -9,23 +9,14 @@
 
     [story]
         [part]
-            story= _ "Dominant Karron heard the Memories Spoken to her by the gliders. She lead several flights west to the Green Isle of the orcs. Striving together with the noble creatures of this far land, the infestation of orcs was destroyed so new eyries could be set there."
+            story= _ "Banished from the Greatland, Dominant Karron led several flights west. The next few flights that arrived on the Greatland reported that some drakes had joined her, planning to build new eyries overlooking the limitless prey of the Green Isle."
```

Comment:
```suggestion
            story= _ "Dominant Karron departed the Greatland following the Spoken Memories of the gliders. On this Windpath, she led several flights far west to ancient lands beyond the ocean. There would they set new eyries overlooking the limitless prey of this remote land."
```

Reply:
Nice. I think "... that remote land.", but either feels good.

Reply:
Please do not reroute Karron to the distant west, because I would like to save-for-later that important first (re-)encounter between the drakes of the distant west and the descendants of the drakes they had exiled to Morogor (i.e. the drakes seen in WoF).

The local culture of drakes (following Gorlack's philosophy and laws) are working to become equal again to the drakes who banished them long ago. This is what the gliders call "reascension" and it basically means criminal rehabilitation of the exiled group's behavior so that they can all someday reunite with the original drake society. But as time goes on, many drakes of other castes increasingly interpret reascension to mean reacquiring just the power and influence of the original far west drake society. So as their influence slowly grows on the GC their culture drifts from 'lawful neutral' to 'lawful evil' (perhaps around the time of SotBE or after).

That would be the time when they reach out to reestablish contact with their honorable "ancestral" culture in the distant west, thinking they are finally worthy of re-acceptance. But instead discover how far they drifted from the original drake ideals (going from lawful good to lawful evil) and are not re-accepted into the fold. Which helps spark an upheaval within the tyrannical drake superpower on the GC, as many drakes there can no longer ignore what it has become (or what they have become).

Keeping most of the original phraseology would be good, too, since by this point in the campaign that is what the player is used to reading. Would the following be acceptable to you @stevecotton -

`"Dominant Karron heard the Memories Spoken to her by the gliders. She lead several flights west to the Green Isle of the orcs. Together with the noble creatures of this far land, she strove to drive off the orcish Horde so new eyries might be set there."`

If you will accept this, then I will accept just dropping the last two lines of the epilogue rather than trying to adjust one of them. At least this way some of the energy of a typical campaign epilogue is preserved, even if it is cut extremely short.

Reply:
There's way too much thought put into things that have absolutely nothing to do with the campaign itself. Why are you pulling out D&D alignments all of sudden? Why are you bringing up SotBE when that has nothing to do with this? The "alignment" of drakes in SotBE (or that time period) is not in scope for WoF. It's not something you can influence.

Reply:
A lot of this seems to be based on the idea that all members of a community will have the same D&D style alignment.

> noble creatures of this far land

What are the criteria for being considered "noble"? From the context, my understanding is that you're describing orcs as an inferior race, and humans as nobility in comparison, despite the larger context that (at least as I read the campaign), Karron will consider humans as prey again as soon as she doesn't need their help against the orcs.

> criminal rehabilitation

Why would anyone expect rehabilitation to be the consequence of exiling a group and leaving that group to educate their own children?

Reply:
> drakes in SotBE (or that time period) is not in scope for WoF.

I am asking for Karron to not go to the distant west in WoF's epilogue. I would just like to leave the option open for the event I described to happen in a future story.

> A lot of this seems to be based on the idea that all members of a community will have the same D&D style alignment.

Naturally, there would be plenty of disagreement between individual drakes just as there are so many heated arguments between the drake characters in WoF over what to do and how to do it. But culture and social inertia can influence enough individuals to cause issues on a larger scale.

> What are the criteria for being considered "noble"?

According to the glider culture it means any type of creature (inclusive of those which cannot speak or craft tools) that were "allies in times of need" to their (pre-exile) ancestral drake society, as recorded in the glider [oral tradition](https://en.wikipedia.org/wiki/Oral_tradition) (i.e. the "Spoken Memories").

A group of humans aided an ancestral group of drakes at some time in the past when their survival was threatened, so the gliders 'remember' this and will advocate for helping the humans in a situation where the roles are reversed (as on the Green Isle where any humans left behind after TRoW will be in desperate condition).

> my understanding is that you're describing orcs as an inferior race,

Really? To me, the newly suggested wording sounds like she is fighting against an epic foe that has superior numbers, Does it not sound that way to you?

> and humans as nobility in comparison, despite the larger context that (at least as I read the campaign), Karron will consider humans as prey again as soon as she doesn't need their help against the orcs.

Part of the idea with describing humans as "noble" is to make it clear they will no longer be considered prey items; they are being granted a much higher status. Particularly now that Karron has finally "heard" the glider "Spoken Memories" (she finally takes the glider viewpoint seriously).

> Why would anyone expect rehabilitation to be the consequence of exiling a group and leaving that group to educate their own children?

The drakes who banished them had no belief they could be rehabilitated. Only the gliders _among the exiled group_ clung to the hope that rehabilitation and reconciliation were still possible for themselves and their fellow exiles.

Reply:
Saying "former allies" might be clearer than "noble creatures", however it still doesn't fit with player knowledge, you can't expect players to interpret things based on stuff in a story bible that's labelled "spoilers". From the events in this campaign, there's no reason to think that there was a previous alliance between humans and drakes; quite the opposite, based on the misnaming of humans as haldric, it seems that drakes haven't met humans before TRoW. Resha herself says "The full ability of the humans is still unknown to us." in S06.

> an epic foe that has superior numbers

But individually, no members who would be described as "noble creatures". In the context, there seems to way for Karron to know many (if any) humans on the Green Isle, which implies the division into "noble" and "non-noble" is happening entirely based on race.

Reply:
> it still doesn't fit with player knowledge,

Players who only played WoF just need to know that Karron has gone off to fight the orcs of the Green Isle alongside other species which she has embraced as allies (a big step up from treating them as prey).

Players who also played TRoW can make an educated guess that human survivors/slaves are a strong candidate as an ally against the orcs, there. Possibly also woses and mermish. But those details are left to the imagination.

> Resha herself says "The full ability of the humans is still unknown to us." in S06.

At that point she was still carefully studying the humans (covertly, from a high altitude) to correctly identify them with one of the creatures described in the glider oral tradition. After she fails to dissuade the sudden attack on the humans, she tries to reveal the full truth in the next scenario (S6x):

`"Nay, they were not drakes which he attacked... Those allies [Dominant] Morogor betrayed were creatures of an alien kind. Perhaps even the same kind our hunters now chase across the grassy flatland. I should have long ago Spoken the whole of this Memory to you. Yet I had concern it was more than you could believe by word alone."`

In the context of the immediately preceding (S6) event and the (S6x) story text, _"the same kind our hunters now chase across the grassy flatland"_ is referring to the humans (of Wesnoth). So she is worried the humans may be an old ally which they are now mindlessly slaughtering.

All conflicts with humans taking place during WoF are the result of early mistakes made by the drakes. So a drake who has learned from those mistakes has no reason to attack humans on the Green Isle, and some very good reasons to ally with them instead.

> In the context, there seems no way for Karron to know many (if any) humans on the Green Isle, which implies the division into "noble" and "non-noble" is happening entirely based on race.

She could and presumably would get to know them after freeing them and fighting beside them for some amount of time. The text does not seem to imply that Karron (or the narrator character) classes them as "noble" upon first sight? The whole line seems vague and is speaking in the past tense (well, except for the missing "lead" -> "led" fix).

Reply:
Although Vladnir confirms Resha's assessment during S6x, he proceeds to die fighting humans later. Even without that detail, I'm unconvinced that a majority of players will join the dots as you do.

However, I think the whole discussion about "noble creatures" is unproductive. Nemaara has provided a new wording, why not work from that as a starting point, instead of trying to keep "noble creatures" in the text?

Reply:
Huh? I'm a little confused? The text I provided doesn't mention making contact with other drakes nor does it even say exactly where they end up (one could even interpret it as them arriving on the Green Isle). 

Another question is if we presume they end up on the Green Isle, should we assume there would be a sizeable human population still remaining? Or would they be too few by that time to help against the Lich Lords/orcs? These kinds of questions or why I prefer leaving it vague unless we have a specific idea of how it should happen.

Reply:
> The text I provided doesn't mention making contact with other drakes nor does it even say exactly where they end up (one could even interpret it as them arriving on the Green Isle).

Well, the _"far west to ancient lands beyond the ocean"_ part implies the western continent, which is where the other drakes banished them from. After all, the Green Isle is _in_ the ocean rather than beyond it. Also, the drakes would not be able to fly far enough to make it past the Green Isle without resting there and acquiring food. They do not have or use ships so they have to island hop to make an oceanic crossing.

> Another question is if we presume they end up on the Green Isle, should we assume there would be a sizeable human population still remaining?

Presumably the same as the protagonist human slave population in NR, working in the slave mines as predicted by Lady Jessene in TRoW S7. The woses of the Oldwood might also be willing to help, once orcs started felling their forest for timber. We also know that a very large contingent of the orcs occupying the Green Isle leave for the Great Continent in search of the Ruby of Fire just prior to LoW. With their numbers thinned out, that would be the moment to make a move. Especially if the silver mage known as "Seeker Othila" (mentioned being left behind on the Green Isle in WoF S5) managed to switch Jevyan's old teleportation gates from a connection with the orcish homeland to a connection with the homelands of the orcs' enemies.

> why I prefer leaving it vague unless we have a specific idea of how it should happen.

See above for a specific idea. but the new wording I suggested is also pretty vague:

`"Dominant Karron heard the Memories Spoken to her by the gliders. She led several flights west to the Green Isle of the orcs. Together with the noble creatures of this far land, she strove to drive off the orcish Horde so new eyries might be set there."`

All it says is where she went and what she is attempting to do there. Whether or not she succeeds is left open.

Reply:
That wording strongly implies that she reached her destination. I think it would be better for it to be more vague – state her intent (to fly west and establish new eyries and defeat the orcs) but try not to imply she succeeded even in arriving.

The word "strove" seems to be the main culprit, I think. Changing that to something with a meaning similar to "planned" is likely sufficient to address this particular concern. (I don't know if there are other concerns it leaves unaddressed.)

Reply:
She does reach her destination; it is the question of whether or not she succeeds at driving away the orcish occupation of the Green Isle which is left vague by the above wording. (And her arrival date is left vague as well.)

Reply:
Incorrect. I don't believe you have the right to decide whether she reaches her destination. It has nothing to do with the actual events of the campaign.

Reply:
Wesnoth epilogues typically tell what happened to the main characters after the events of the story- where they go, what they do and/or what they become., This gives players a sense of closure about them.

The Green Isle is the ending that fits Karron's underlying character motivation and which the story sets up for her (or tries to). She desires the comradery of being a part of a team that is taking on huge challenges, but her pride often gets in the way of that. After suffering defeats, she is humbled enough to finally consider a significant change of approach. And that change of approach allows her to actually fulfill her underlying goal- she becomes part of a team (of drakes and other species) taking on the big challenge of pushing back an orcish occupation from the Green Isle.

Which also mirrors (so as to help vaguely setup) the role drake characters like Krash play in NR; helping human slaves free themselves from their orcish masters.

So if @nemaara will approve my text change suggested [above](https://github.com/wesnoth/wesnoth/pull/8124#discussion_r1441229388) then I will consider it a fair enough compromise and not ask for any more changes.

Reply:
Is Green Isle mentioned anywhere else in this campaign?

Reply:
> This gives players a sense of closure about them.

If it's about the focal character (the leader), that might be okay, but Karron is not the focal character, right? A sense of closure in an open-ended moddable game like Wesnoth is generally not a good thing.

Reply:
> Is Green Isle mentioned anywhere else in this campaign?

It's implied repeatedly, because TRoW's voyage landed on the isle that S02 is set on, and the drake's path then rests on the same other islands that TRoW did, along with tying into the aftermath of TRoW when they reach the Great Continent. It's only mentioned a couple of times by name, in fact Haldric gets name-dropped more often that Green Isle, but there's little doubt that drakes going west from Morogor would find the Green Isle as the next place where they can land.

Reply:
I'd rather not name drop it if we didn't mention it anywhere else, or give it a different name that the drakes would know about. Why would they call it the Green Isle? And do they know that there are orcs there? We the devs know that from TRoW but why would they the drakes know anything about it? A player who hasn't played TRoW also wouldn't know anything about it. That's part of the reason why I wanted to make it more vague (because from the perspective of the drake narrator, they probably don't know much on that side, unless you're telling me that they explored it before).

Reply:
The whole of S02 is about the ships traveling between the Green Isle and Great Continent, with the aim of capturing the enemy leader alive for interrogation. Resha says the name while giving the results of the interrogation, so that line makes sense, although it does assume that the Orcish name for it is the Green Isle; I suppose that came from the Lich Lords.

The drakes probably have a temporary name for it while only knowing that ships come from there, assuming that the Secrets or Spoken Memories haven't provided one. If it appears at all it will need to be clear enough to be recognised around 10 scenarios after it was last mentioned, so I think it's best to leave it out.

"Dominant Karron departed the Greatland following the Spoken Memories of the gliders. On this Windpath, she led several flights to seek the lands west beyond Morogor. There would they set new eyries overlooking the limitless prey of that remote land."

Reply:
> Resha says the name while giving the results of the interrogation, so that line makes sense, although it does assume that the Orcish name for it is the Green Isle; I suppose that came from the Lich Lords.

Also, Resha had at least one [conversation](https://github.com/wesnoth/wesnoth/blob/c255620b76a0462c34d16fcf42a798c6e95f0679/data/campaigns/Winds_of_Fate/scenarios/02_Reclamation.cfg#L475) with the Crown Prince of Southbay (Southbay was a city on the Green Isle) during one of his and Lady Jessene's expeditions across the ocean in the years before TRoW. So she presumably would have heard (or overheard) some basic facts about them, such as the name of their homeland (even if other details were lost in translation, like the name of the human species).

> The drakes probably have a temporary name for it while only knowing that ships come from there,

The drakes only made up exonyms for things they did not know the proper names for. So they refer to Wesmere by its real name since the saurians told them its real name (though drakes did not yet have a concept of a temperate forest biome so they conceptualize it as a "cold" jungle instead). Their interrogation of the orcish sea captain revealed the name of the Green Isle so they just call it the Green Isle.

>  If it appears at all it will need to be clear enough to be recognised around 10 scenarios after it was last mentioned,

I think as long as players understand she followed the suggestion to "take her flight west beyond even Morogor" by going to someplace named "the Green Isle", then they will not be confused. At worst, they would assume "Green Isle" is just another drake exonym for whichever land she ended up settling on.

And for players who have played or will later play TRoW, this tie in gives them some possibilities to ponder, regarding whether there is any hope for that land and its remaining people, which they were forced to leave behind (during TRoW).

> A sense of closure in an open-ended moddable game like Wesnoth is generally not a good thing. I understand why you'd want it – we all like to have closure on things – but in this case it doesn't suit the medium.

I do agree in general, but for players to feel satisfied, I have to hit the right note between open-ended wonder (about the wider story world) and some amount of closure (especially for the main characters). I am trying to leave it open ended as to whether she succeeds or fails, lives or dies, while still providing some degree of closure for her arc.

Reply:
> The drakes only made up exonyms for things they did not know the proper names for.

This feels like slightly unnatural behaviour, to be honest. If it's a location that's not especially important to them, then maybe it makes sense to use the denizens' name for it, but even then, it's not unusual to make up a name, or misinterpret the denizens' name. That's where names like "Canada", "Peking", "Bombay", and "Persia" come from, after all. But if it's a location that's important to them, they'd probably come up with their own name for it. I don't know if the Green Isle fits that second category, mind you; it just sounds like it might from the discussion so far.

> I think as long as players understand she followed the suggestion to "take her flight west beyond even Morogor" by going to someplace named "the Green Isle", then they will not be confused. At worst, they would assume "Green Isle" is just another drake exonym for whichever land she ended up settling on.

If you want to leave open the possibility that the drakes' "Green Isle" might not be the same as the Wesfolk's "Green Isle," I'd suggest making the name slightly different. Even just "Green Island" might work.

> I do agree in general, but for players to feel satisfied, I have to hit the right note between open-ended wonder (about the wider story world) and some amount of closure (especially for the main characters). I am trying to leave it open ended as to whether she succeeds or fails, lives or dies, while still providing some degree of closure for her arc.

I'm glad you seem to at least somewhat get my point. I don't think I agree that declaring she successfully arrived is a necessary level of closure though.

Other than my previous suggestion, based on Steve Cotton's later proposal, I think just adding "would" would be enough. Instead of "she strove to drive off", say "she would strive to drive off". Though I also like Steve Cotton's version.

Reply:
I get the point, so having them mention Green Isle has thought behind it but we will need to do something about name dropping it more in 1.19. Humans maybe could have some text about it, or add some more of Resha's memories regarding it. 

For the exact wording, how about "Dominant Karron heard the Memories Spoken to her by the gliders. She led several flights west beyond Morogor to the distant Green Isle. There would she she strive to drive off the prey of this far land so new eyries might be set there."

I prefer not saying just orcs because there may be undead and remaining lich lords. We also don't know exactly which factions on the Green Isle the drakes would ally with, be enemies with, or be neutral with. If you want to add noble creatures that is okay with me but it makes the line kind of long doesn't it?

Reply:
"drive off the prey" feels a little bit odd, it's mixing them being a competitor vs being a food source. Is the primary motivation to secure sufficient food, with the removal of competition a secondary motivation, or is it the other way round?

I'd still prefer a variant of "overlooking bountiful plains", and leave the competition merely implied, but how about "There would she strive for dominion over the bountiful land, driving competing prey far from the new eyries."?

Reply:
>  we will need to do something about name dropping it more in 1.19. Humans maybe could have some text about it

I should be able to get the humans in the beginning of S6 to mention the Green Isle in a way that sounds natural, given the subject matter of their conversation.

Also, I forgot to mention this before, but the Green Isle is also name dropped in S5's [log book](https://github.com/wesnoth/wesnoth/blob/441a2cf5e46057d1166b6e994d9052dbf2838357/data/campaigns/Winds_of_Fate/scenarios/05_Threshold.cfg#L720).

> I prefer not saying just orcs because there may be undead and remaining lich lords.

I hope there are remaining lich-lords, since that would make the situation a lot more interesting than just orcs by themselves. But something that surfaced in this conversation is that 1.18's LoW has no depiction or even mention of a lich-lord. The orcs are out looking for the Ruby of Fire by themselves, rather than being overseen by a lich-lord on such an important task. Which somewhat implies there are no more lich-lords or they have lost control over their orcs.

Anyway, the reason there is no mention of undead is because drakes around the time of WoF seem to not recognize undead as what they actually are or how powerful necromancy is (drake understanding of magic is very poor in general) and they tend to lump undead together with living humans. As hinted at in S8. Also, they are not against cooperating with them as seen in S8 and suggested later by a [lore book](https://github.com/wesnoth/wesnoth/blob/441a2cf5e46057d1166b6e994d9052dbf2838357/data/campaigns/Winds_of_Fate/scenarios/11_Crosswind.cfg#L554) in S11 where lich-lord Lenvan is briefly mentioned as some kind of advisor or ally at one point in the ancient past.

So the idea here is the drake narrator is either miscategorizing the lich-lords/undead as "noble creatures" (i.e. just another form their human allies can take) or misidentifying them as some auxilia of the orcish horde (akin to trolls, wolves or naga) rather than its leaders.

Errors like this are a running theme throughout the story, including the narration. What I think is missing in this case is a code comment clarifying what the actual truth is supposed to be, like there are in other parts of the campaign.

So:

`# Which "noble creatures" Karron might ally with (against the orcs) likely includes humans slaving in the mines for their orcish masters (similarly to NR) and the woses of the Oldwood Forest.`

`# The drake narrator fails to mention any surviving lich-lords since their power stems from magic, which drakes have very little knowledge of and may still tend to underestimate.`

`"Dominant Karron heard the Memories Spoken to her by the gliders. She led several flights west to the Green Isle of the orcs. Together with the noble creatures of this far land, she strove to drive off the orcish Horde so new eyries might be set there."`

Reply:
That wording is again putting "noble creatures" and "orcish" into the same sentence, in a way that implies the definition of "noble" mostly involves "not being an orc". No matter how much discussion happens in this thread, the average player will be reading it based solely on the text and events in the campaign.

Why is the text specifically about orcs anyway? If a group of humans behaves the same as a group of orcs in terms of its treatment of the wildlife, why would the drakes treat them differently? Why is the text about orcs at all, instead of looking at food supply and general threats to that food supply?

> She could and presumably would get to know them after freeing them and fighting beside them for some amount of time. The text does not seem to imply that Karron (or the narrator character) classes them as "noble" upon first sight?
>
> So the idea here is the drake narrator is either miscategorizing the lich-lords/undead as "noble creatures" (i.e. just another form their human allies can take) or misidentifying them as some auxilia of the orcish horde

These two explanations seem to contradict each other - one that they're noble based on knowledge, the other that they're noble based on lack of knowledge. Both for this review and for the translations, please give a clearer definition of what you're trying to say with "noble". I've suggested "former allies" as an alternative, along with reasons why I don't think that interpretation will fit with player knowledge from the campaign.

Reply:
> No matter how much discussion happens in this thread, the average player will be reading it based solely on the text and events in the campaign.

Why would we assume the average player will have only played this one campaign? What is the point of having continuity between campaigns if we assume that?

> Why is the text specifically about orcs anyway?

Because orcs:
* Are the dominant force on the Green Isle (as far as drakes can tell) which is the next available land west of Morogor.
* Would not tolerate drakes any more than the native humans which orcs just finished genociding and enslaving.
* Invaded Morogor and heavily damaged one of its precious remaining ecosystems.
* Are recorded in the glider and clasher histories as having posed existential threat(s) in the past also.
* Are a powerful enough threat to represent the kind of challenge Karron feels she needs to overcome in her life.

> If a group of humans behaves the same as a group of orcs in terms of its treatment of the wildlife, why would the drakes treat them differently?

They would treat them the same way, of course. But why would you expect drakes to conclude that in a time period where humans are an endangered species? Humans are still hundreds or thousands of years away from dropping a sun on the world and drakes cannot see the future.

> Why is the text about orcs at all, instead of looking at food supply and general threats to that food supply?

Because most of this campaign's story was about drakes moving past seeing everything in those simplistic terms?

In the beginning of the story, many drakes saw themselves the way we real life humans see ourselves on Earth. "We are the uncontested rulers of the world and other species exist as our resources." But by the end they learn this is very much not a realistic viewpoint. Many other species are equal to or much stronger than them. If they want to survive then they need to stop seeing themselves as the center of the universe.

> These two explanations seem to contradict each other - one that they're noble based on knowledge, the other that they're noble based on lack of knowledge.

As per preexisting canon and the campaign's story bible, I am leaving the (Morogor) drakes ignorant to magic and its true power.

This allows drakes to play some misguided role supporting Wesnoth's creation of the artificial suns that cause The Fall (as revealed in UtBS).

> Both for this review and for the translations, please give a clearer definition of what you're trying to say with "noble".

It is a subjective drake term, which basically means another species (or perhaps even a member of another species in some cases) which generally behaves (close enough to) a drake's sense of "honor" and which does not pose a clear and serious threat to drake survival (at least as far as they can tell) but instead acts or acted as an ally in a serious crisis. Who and what qualifies as "noble" will vary between individual drakes and may evolve quite a lot over the centuries to come.

Reply:
> * Would not tolerate drakes any more than the native humans which orcs just finished genociding and enslaving

@nemaara: Where is the moderator when people are using triggering language to misrepresent events in world of wesnoth?

Reply:
> Why would we assume the average player will have only played this one campaign? What is the point of having continuity between campaigns if we assume that?

Okay, they're likely to also know the events in TRoW, and maybe join up the dots with LoW. But that leaves the fourth and fifth points here:

> Because orcs:
> 1. Are the dominant force on the Green Isle (as far as drakes can tell) which is the next available land west of Morogor.
> 2. Would not tolerate drakes any more than the native humans which orcs just finished genociding and enslaving.
> 3. Invaded Morogor and heavily damaged one of its precious remaining ecosystems.
> 4. Are recorded in the glider and clasher histories as having posed existential threat(s) in the past also.
> 5. Are a powerful enough threat to represent the kind of challenge Karron feels she needs to overcome in her life.

1. I think the majority decision is to write the epilogue as her future plans while she's travelling there. What does she know at that point, other than third-hand knowledge from the captured ship's captain?
3. Morogor's ecosystem was breaking from overpopulation and rising sea levels before the orcs arrived.
4. How is the average player meant to know that?
5. How is the player meant know that? While it would be a motivation, how would someone still be in military command if they based their decisions on that? Does it demonstrate that the character has learnt and grown during the story?

For point 2., genocide is something specific, more than just taking land and power; is there a reason in TRoW to think that the orcs were after more than land, power and survival? Sure, the orcs are likely to be continually at war with the drakes over resources and control of the land, but that can be covered in the epilogue by saying that Karron is after control of land, rather than prioritising killing orcs.

> > If a group of humans behaves the same as a group of orcs in terms of its treatment of the wildlife, why would the drakes treat them differently?
> 
> They would treat them the same way, of course. But why would you expect drakes to conclude that in a time period where humans are an endangered species? Humans are still hundreds or thousands of years away from dropping a sun on the world and drakes cannot see the future.

Consider a group of bandits, living off the land and moving on when the local animals get too scarce, or when the orcs come looking for them. What, if anything, differentiates them from a stereotypical group of orcs?

I'd expect the drakes to notice that these humans weren't farming, and that wildlife around them was scarce.

> > Why is the text about orcs at all, instead of looking at food supply and general threats to that food supply?
> 
> Because most of this campaign's story was about drakes moving past seeing everything in those simplistic terms?

What's simplistic about food? The growth and logistics of a secure food supply aren't simple, and securing enough land to have a secure food supply seems a logical military objective.

Claiming that killing or driving off all members of a competing race will make the food supply secure sounds simplistic, if it's based on the assumption that removing them will in and of itself make the food supply secure. There's a lot of real-world politics that show it's a vote winner, as long as those politicians don't get into actual power and have to implement it, which is part of the reason that I'm pushing back against having it as the successful epilogue. It works as the plot for a villain, or a backstory for a hero to arise and fix, but not as a heroic ending.

> It is a subjective drake term, which basically means another species (or perhaps even a member of another species in some cases) which generally behaves (close enough to) a drake's sense of "honor" and which does not pose a clear and serious threat to drake survival (at least as far as they can tell) but instead acts or acted as an ally in a serious crisis. Who and what qualifies as "noble" will vary between individual drakes and may evolve quite a lot over the centuries to come.

So generally declaring some races as "noble", and teaming up with them to kill all members of another race. Do you understand why some of us are feeling uncomfortable parallels to real-world racism, to the extent that we're actively moving to change the text that we collectively publish?

Reply:
Well we have 10 days for the next release so I think it's close to time for me to make a decision. I was hoping we'd be able to work it out to satisfy everyone but obviously that isn't the case. So I propose we remove the mention of orcs altogether but keep the mention of noble creatures as that is part of drake language and might be how they refer to potential allies. So it'd be something like

"Dominant Karron heard the Memories Spoken to her by the gliders. She led several flights west beyond Morogor to the distant Green Isle. Together with the noble creatures of this far land, she strove to drive off the prey there so new eyries might be set there."

And this way it's not targeting any particular race and leaves it to our imagination what the factions end up being. Orcs aren't a monolith either, for all we know they could fight against each other and some would be willing to ally with Karron and co.

Reply:
Could maybe remove the first "there": "…strove to drive off the pray so new eyries might be set there."

I would also prefer to use conditional tense ("would strive").

Reply:
Okay let's use celmin's version.

"Dominant Karron heard the Memories Spoken to her by the gliders. She led several flights west beyond Morogor to the distant Green Isle. Together with the noble creatures of this far land, she would strive to drive off the prey so new eyries might be set there."

Reply:
> I was hoping we'd be able to work it out to satisfy everyone but obviously that isn't the case.

That is a strange and disappointing statement for you to make.

- if the fundamental disagreement is about the explicit expression of hateful, racist views being acceptable in mainline wesnoth, what would a trade-off look like? What is the middle ground between yes and no? Would that answer be that racism against southern people is not acceptable but actually OK against people from East?
- you seems reluctant to make a decision, but this is actually the main responsibility of the project title that you carry to get involved in such debate and make a official statement that represents the project. You could have decided to get involved in the discussion before in order to spare some dreadful and exhausting discussions for everyone.
- the idea that wesnoth containing explicit depiction of genocide and slavery is not a matter that you are interested in getting involved with is disappointing at best. Are average contributors set to have this type of exchanges more in the future because the lore director doesn't want to make their mind?

At this time, the only general statement of what is acceptable in wesnoth is [basically](https://github.com/wesnoth/wesnoth/issues/7361#issuecomment-1429035958) "anything that exists in the real world or worse". This is at the same time a betrayal of wesnoth tradition ([WINR](https://wiki.wesnoth.org/Glossary), NRIW...), stupid (who wants to get involved to a medieval fantasy world that is an exact copy of the real world), and scary (should we get ready for graphic depiction of torture, rape, feminicide and utopian incels' society?). Can't you guys do better than that?

Reply:
@nemaara the last sentence does not make any sense inside or outside of the story; predators driving away the prey means they would starve. Would you at least replace "the prey" with "one of the ancient foes" or simply "the Horde" (no mention of orcs so players can decide for themselves if this means lich-lords, orcs or something else like the monster that can be disturbed awake in S5.

Reply:
>     1. I think the majority decision is to write the epilogue as her future plans while she's travelling there.
>     2. Morogor's ecosystem was breaking from overpopulation and rising sea levels before the orcs arrived.
>     3. How is the average player meant to know that?
>     4. How is the player meant know that? While it would be a motivation, how would someone still be in military command if they based their decisions on that? Does it demonstrate that the character has learnt and grown during the story?

1. A totally arbitrary restriction from a voting population of 3.
2. Is stealing food from a starving person less terrible than stealing from one who is well fed?
3. The opening story briefing of S2.
4. The war criminal behavior of the orcish invasion as depicted in TRoW matches their own historical accounts.

> For point 2., genocide is something specific, more than just taking land and power; is there a reason in TRoW to think that the orcs were after more than land, power and survival?

It is the deliberate destruction of a people. What fate do you think everyone is running away from in TRoW? What is being depicted in its story images? Do you think Jessene is lying when she says the orcs will not tolerate living humans outside of the slave mines?

Compare that to a group of drakes attempting to "drive off" (either back through the gates or via their fleet) that same orcish army with the help of surviving victims.

> Consider a group of bandits, living off the land and moving on when the local animals get too scarce, or when the orcs come looking for them. What, if anything, differentiates them from a stereotypical group of orcs?

Numbers. If a hundred years after the orcs are driven back, a large population of wesfolk poachers has grown to a similar size and violates their hunting grounds at the same scale the orcs did, why would you not expect them to get the same treatment?

> What's simplistic about food?

Seeing everything that moves as a food source is too simplistic. Recognizing a natural ally/neutral like the dwarves who mostly do not compete for the same resources (above ground resources) or the dangerous predator that most orc clans represent to other species is critical to their survival beyond what they were used to on Morogor.

> There's a lot of real-world politics that show it's a vote winner

Yes, I am sure many real world politicians want to be associated with an ominous swarm of misguided people eating dragons that get themselves into so many troubles at once they have to head for the hills to survive the natural backlash they caused. Sounds like a great campaign platform.

> not as a heroic ending.

Protagonists are not heroes; for example see the antivillain protagonists of SotA, DiD and later parts of TRoW (Haldric killing whoever the elves direct him to in exchange for land) and even HttT (Konrad telling Li'sar: `"(hushed) Our soldiers will defend us as we rush across the field. Sorry to be blunt, but you and I are the only ones who need to make it across alive."`).

Like SotA and DiD, WoF's protagonists are probably antivillains.

> Claiming that killing or driving off all members of a competing race

Alright, this is a total misrepresentation of what any currently proposed version says - the particular group of orcs on the Green Isle five years after their invasion are not "a race", they are an occupying army. They are a band of colonizers that invaded a foreign land, killed most of the natives and enslaved the rest. You keep trying to paint them as the victims and/or as some immobile nonmilitary population.

Would you accept more specific and clear wording like "drive off the orcish host"? (Where "host" means "army" in archaic English.)

Reply:
I think "drive off the orcish host" is a great idea - it's both more specific AND it fits the Wesnoth universe with its archaic wordings appearing from time to time

Reply:
I think I'm okay with "drive off the orcish host"…

Reply:
I forgot that Markdown changes numbers - I moved 2 out to answer it separately, and missed that my 3, 4 and 5 were renumbered. Please reread that as:

`3.` Morogor's ecosystem was breaking from overpopulation and rising sea levels before the orcs arrived.

`5.` How is the player meant know that? While it would be a motivation, how would someone still be in military command if they based their decisions on that? Does it demonstrate that the character has learnt and grown during the story?

> > For point 2., genocide is something specific, more than just taking land and power; is there a reason in TRoW to think that the orcs were after more than land, power and survival?
> 
> It is the deliberate destruction of a people. What fate do you think everyone is running away from in TRoW? What is being depicted in its story images? Do you think Jessene is lying when she says the orcs will not tolerate living humans outside of the slave mines?
>
> Compare that to a group of drakes attempting to "drive off" (either back through the gates or via their fleet) that same orcish army with the help of surviving victims.

Remember, this discussion started with "the infestation of orcs was destroyed". The motivations for the actions affect which crime it is, and the motivations also imply how far the action would be taken, and how cruelly it would be done. From the wording and motivations assigned to Karron, even in some of the "drive off" suggestions, she often seems to be killing orcs out of hatred, which implies that she would pursue every orc on the Green Isle, even if they took up agriculture or sustainable farming.

While slavery is always a terrible crime, there's still a difference between using it as a way to get cheap labor, and using it as a method of slow execution. I'm assuming the orcs are doing it for cheap labor.

> Numbers. If a hundred years after the orcs are driven back, a large population of wesfolk poachers has grown to a similar size and violates their hunting grounds at the same scale the orcs did, why would you not expect them to get the same treatment?

I would expect them to get the same treatment, my question is why would that hundred-year gap be significant? Given a group of humans and a group of orcs behaving similarly, why do the humans get a grace period?

> Yes, I am sure many real world politicians want to be associated with an ominous swarm of misguided people eating dragons that get themselves into so many troubles at once they have to head for the hills to survive the natural backlash they caused. Sounds like a great campaign platform.

From the context, it's clear that I was talking about humans vs humans. There's a lot of real-world politicians who won votes by claiming that deporting, indirectly killing or sometimes even directly killing a particular race, or immigrants in general, will in and of itself make the food supply secure. Or in modern times, make access to jobs and healthcare secure; and people will vote for them, even when told that cutting immigration will mean less healthcare staff.

> Alright, this is a total misrepresentation of what any currently proposed version says - the particular group of orcs on the Green Isle five years after their invasion are not "a race", they are an occupying army. They are a band of colonizers that invaded a foreign land, killed most of the natives and enslaved the rest. You keep trying to paint them as the victims and/or as some immobile nonmilitary population.

There's a concept of collective punishment there. I'm assuming that whole tribes came through the portals, so there's a generation growing up who weren't the invasion force.

> Would you accept more specific and clear wording like "drive off the orcish host"? (Where "host" means "army" in archaic English.)

That's still obviously based on race, and it's not obvious that it's limited to the military.

How about a different direction to the text?

#po: The battle plan echoes Gorlack’s plan from S02.
"Dominant Karron heard the Memories Spoken to her by the gliders. With maps of the Green Isle that the Saurians had taken from Elensefar, she led several flights west beyond Morogor. Quietly would each flight set their chosen eyrie. One by one would the eyries reveal themselves, and when the enemy warlords encircled the new eyrie, then would the other wings strike at the exposed backs of the encirclement."

Reply:
I don't think a battle plan seems right in the context of the epilogue. I can accept using the phrase the Horde so something like this 

"Dominant Karron heard the Memories Spoken to her by the gliders. She led several flights west beyond Morogor to the distant Green Isle. Together with the noble creatures of this far land, she would strive to drive off the opposing hordes so new eyries might be set there."

Pick your fav adjective instead of opposing like assailing or enemy or adversary. 

Also who is demario?

Reply:
> "Dominant Karron heard the Memories Spoken to her by the gliders. She led several flights west beyond Morogor to the distant Green Isle. Together with the noble creatures of this far land, she would strive to drive off the opposing hordes so new eyries might be set there."

Okay, this works well enough.

Reply:
While I'm not completely comfortable with the latest revision, it does seem a major improvement on the existing text. I'd like to accompany it with a comment to clarify for translators that "noble creatures" doesn't mean a race-based divide.
```
#po: "noble creatures" is a drake term for allies and potential allies
```

In the context of Wesnoth, I'd generally assume "hordes" implied "orcs", but it's got some ambiguity to it. I'd prefer to use a synonym, but "armies", "forces" or "legions" don't seem to fit as well.

Reply:
If I had to pick one of those three, "legions" is the least bad.

Reply:
> While I'm not completely comfortable with the latest revision, it does seem a major improvement on the existing text. I'd like to accompany it with a comment to clarify for translators that "noble creatures" doesn't mean a race-based divide.
> ```
> #po: "noble creatures" is a drake term for allies and potential allies
> ```
> 
> In the context of Wesnoth, I'd generally assume "hordes" implied "orcs", but it's got some ambiguity to it. I'd prefer to use a synonym, but "armies", "forces" or "legions" don't seem to fit as well.

Let's go with your version incl the po comment, please pick your favorite synonym as you like. Hosts, swarms, throngs in addition to the above can all work.